apps/els_core/src/els_config.erl

```diff
@@ -92,27 +92,37 @@
 
 -spec initialize(uri(), map(), map()) -> ok.
 initialize(RootUri, Capabilities, InitOptions) ->
-    initialize(RootUri, Capabilities, InitOptions, _ReportMissingConfig = false).
+    initialize(RootUri, Capabilities, InitOptions, _ErrorReporting = no_els_server).
 
--spec initialize(uri(), map(), map(), boolean()) -> ok.
-initialize(RootUri, Capabilities, InitOptions, ReportMissingConfig) ->
+-spec initialize(uri(), map(), map(), use_els_server | no_els_server) -> ok.
```

Comment:
My reasoning for changing this parameter was that until I happened upon https://github.com/erlang-ls/erlang_ls/issues/1060 I thought the flag was only about verbosity or the config file missing. But after seeing that issue, it appears the purpose of the flag was to prevent `els_server:send_notification` from being called when the DAP script calls `els_config:initialize/3`.

(Perhaps `els_config:initialize/3` can be removed completely and `els_dap_general_provider.erl` updated to call /4 with `no_els_server`).

Other names for the flag could maybe be `no_notice | show_notice`.